internal/resources/events.go

```diff
@@ -27,4 +27,7 @@ const (
 	ReasonCreatedPersistentVolumeClaim = "CreatedPersistentVolumeClaim"
 	ReasonCreatedDeployment            = "CreatedDeployment"
 	ReasonCreatedStatefulSet           = "CreatedStatefulSet"
+	// ReasonInvalidConfiguration indicates a failure to proceed due
+	// to an invalid configuration.
```

Comment:
I was going to reply that I agree that the wording of the comment was a bit redundant but we needed it because it's an exported type. But when I looked at the file I see that the other constants don't have one. So I'm not entirely sure why I thought it was needed but the other commands don't.  I'll look into it.

Reply:
Fixed.